data/lib/src/datasource_impl/shared_space_document_datasource_impl.dart

```diff
@@ -229,4 +230,24 @@ class SharedSpaceDocumentDataSourceImpl implements SharedSpaceDocumentDataSource
 
     return streamController.stream.first;
   }
+
+  @override
+  Future<WorkGroupFolder> createSharedSpaceFolder(
+    SharedSpaceId sharedSpaceId,
+    CreateSharedSpaceNodeFolderRequest createSharedSpaceNodeRequest) {
+    return Future.sync(() async {
+      final workGroupNode = await _linShareHttpClient.createSharedSpaceNodeFolder(sharedSpaceId, createSharedSpaceNodeRequest);
+      return workGroupNode.toWorkGroupFolder();
```

Comment:
why we need to transfer to folder here?

Reply:
I just return what the API returns to fit with backend